### PR TITLE
Extractor conditonal deploy

### PR DESCRIPTION
**Pull request by thwalker6**
Resolves # None.  An idea I came up with

**Description-**
Its very annoying working with the text extractor.  Commenting and uncommenting out code on deploy is tedious and just awful.  A much simpler solution is to have a conditional on a github action that determines if it should run.  
On line 84 on deploy-experimental it just has if: false. Change it to if: true if you want a text extractor.
Then on line 122 change the needs to be the commented out portion.  Needs cant actually be a conditional. Thanks github.

**Steps to manually verify this change...**

1. The intieal deployment deploys sucessfully.
2. Change the deploy to true for deploying the text extractor on line 84 and on 122 change it to needs: [deploy-static, deploy-text-extractor]
3. A text extractor will deploy to the experimental.
4. Closing the PR will conditionally delete the text extractor based on if one exist.

Notes.  Once a text extractor is deployed you can no longer revert back without closing the PR.  The way cloud formation looks up the extractor arn, it will find it until the extractor is deleted in aws.

### CODE REVIEW

File: .github/workflows/deploy-experimental.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
 
+
```
Comment (peggles2) [inline .github/workflows/deploy-experimental.yml:7]: We can remove this extra line.

File: solution/backend/serverless-experimental.yml [hunk 2]
```diff
@@ -48,7 +48,7 @@ provider:
     DEPLOY_NUMBER: ${env:RUN_ID}
     EUA_FEATUREFLAG: ${ssm:/eregulations/eua/featureflag}
     # Used for when you want to deploy your own
```
Comment (peggles2) [inline solution/backend/serverless-experimental.yml:50]: is this comment needed anymore?

**Review by peggles2 — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-experimental.yml (hunk 1 above)
Comment (thwalker6) [reply .github/workflows/deploy-experimental.yml:7]: removed

File: solution/backend/serverless-experimental.yml (hunk 2 above)
Comment (thwalker6) [reply solution/backend/serverless-experimental.yml:50]: Yes its good to have a comment for what that specifici arn is for.

**Review by peggles2 — APPROVED** [review]

**Pull request merged**